src/Package/Impl/DataInspect/VariableGridWindowPane.cs

```diff
@@ -25,5 +27,31 @@ internal sealed class VariableGridWindowPane : RToolWindowPane {
 
             _gridHost.SetEvaluation(evaluation);
         }
+
+        #region IVsWindowFrameNotify3
+
+        public int OnClose(ref uint pgrfSaveOptions) {
+            _gridHost?.OnClose();
+
+            return VSConstants.S_OK;
+        }
+
+        public int OnDockableChange(int fDockable, int x, int y, int w, int h) {
+            return VSConstants.S_OK;
+        }
+
+        public int OnMove(int x, int y, int w, int h) {
+            return VSConstants.S_OK;
+        }
+
+        public int OnShow(int fShow) {
+            return VSConstants.S_OK;
```

Comment:
Does the reverse need to happen in OnShow?  When you re-open the tool window after closing it, does scrolling work again?